components/navigation/ui/src/main/resources/PhenoTips/LiveTableResults.xml

```diff
@@ -474,8 +476,46 @@
     ##
     ## Object properties from now on
     ##
-    ## Numbers, exact match
-    #elseif($propType == 'NumberClass' || $propType == 'BooleanClass')
+    ## Numbers, exact match or min/max filters
+    #elseif($propType == 'NumberClass')
+      #livetable_getTableName($colname)
+      #set($fromSql = "${fromSql}, ${tableName} ${safe_tableAlias}")
+      #set($whereSql = "${whereSql} and ${targetedObject}.id=${safe_tableAlias}.id.id and ${safe_tableAlias}.id.name = ?")
+      #set($discard = $whereParams.add("${colname}"))
+      ## Converts each used value into correct type
+      #set($numberType = $propClass.get($colname).getProperty('numberType').getValue())
+      #if($numberType == 'integer')
+        #set($typedFilterValue = $mathtool.toInteger($filterValue))
+        #set($typedFilterValueLowLimit = $mathtool.toInteger($filterValueLowLimit))
```

Comment:
If the user enters min = `5.5` for an `integer` or a `long` property, this code will include the results where the property value is `5`. This is counterintuitive for users. We can't expect them to know the property is an integer and they're only supposed to enter integer values in the filters. In my opinion min/max filters should *not* be rounded down to the nearest integer in any of the 4 cases.